### PR TITLE
Create 2024-03-21-uswds-monthly-call-march-2024.md

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary
Add USWDS event page for March 21, 2024.

- [ ] update `primary_image` with https://github.com/GSA/digitalgov.gov/pull/7424

### Preview
[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-create-march-uswds-call/event/2024/03/21/uswds-monthly-call-march-2024/)

### CODE REVIEW

File: content/events/2024/03/2024-03-21-uswds-monthly-call-march-2024.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+*This event is part of a monthly series that takes place on the third Thursday of each month. Don’t forget to set a placeholder on your personal calendar for our future events this year.*
+
+## About the USWDS
+
+[The U.S. Web Design System](https://designsystem.digital.gov/) is a toolkit of principles, guidance, and code to help government teams design and build accessible, mobile-friendly websites backed by user research and modern best practices.
+
+* [The U.S. Web Design System](https://designsystem.digital.gov/)
+* [Contribute on GitHub](https://github.com/uswds/uswds/issues)
+* [Email Us](mailto:uswds@support.digitalgov.gov)
```
Comment (bonnieAcameron) [inline content/events/2024/03/2024-03-21-uswds-monthly-call-march-2024.md:54]:
```suggestion
* [Email Us](mailto:uswds@gsa.gov)
```

**Review by bonnieAcameron — CHANGES_REQUESTED** [review]
The email to uswds was out of date, other than that, looks good

File: content/events/2024/03/2024-03-21-uswds-monthly-call-march-2024.md (hunk 1 above)
Comment (brunerae) [reply content/events/2024/03/2024-03-21-uswds-monthly-call-march-2024.md:54]: Thank you, Bonnie!!

**Review by bonnieAcameron — APPROVED** [review]

**Review by brunerae — APPROVED** [review]
Looks great! I reviewed the content against our event brief and found no issues.

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**